Tests/SwiftParserTest/AttributeTests.swift

```diff
@@ -422,12 +422,23 @@ final class AttributeTests: XCTestCase {
 
     assertParse(
       """
-      @_expose(Cxx, 1️⃣baz) func foo() {}
+      @_expose(Cxx, 1️⃣baz2️⃣) func foo() {}
       """,
       diagnostics: [
-        DiagnosticSpec(message: "expected string literal to end @_expose arguments"),
-        DiagnosticSpec(message: "unexpected code 'baz' in attribute"),
-      ]
+        DiagnosticSpec(
+          locationMarker: "1️⃣",
+          message: #"expected '"' in string literal"#,
+          fixIts: [#"insert '"'"#]
+        ),
+        DiagnosticSpec(
+          locationMarker: "2️⃣",
+          message: #"expected '"' to end string literal"#,
+          fixIts: [#"insert '"'"#]
+        ),
+      ],
```

Comment:
How difficult is it to make this a single diagnostic? Seems a little weird as two to me, ie. I would expect a `surround '...' in '"'`

Reply:
https://github.com/apple/swift-syntax/issues/1691